docs/content/guides/getting-started/introduction.md

```diff
@@ -27,24 +27,24 @@ Explore Handsontable's features and source code:
 ## Guides for your framework
 
 <div class="row-items-container">
-  <Link href="../javascript-data-grid/" hide-latest-version class="row-item">
-    <Img class="integration-framework-logo" src="$withBase('/img/pages/introduction/javascript.svg')" alt="JavaScript logo" />
+  <Link href="/docs/{docsVersion}/binding-to-data/" hide-latest-version class="row-item">
```

Comment:
These links point to the `react-data-grid` pages, while they should point to `javascript-data-grid`. (apart from the React one)

Reply:
I reverted the changes within f1b38a15b2e707982d4c822fe671ace4ebfd0e3e. It should be fixed.